Chapter-1-initial-architecture/Src/Fitnet.ArchitectureTests/Contracts/ContractsModuleArchitectureTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.ArchitectureTests.Contracts;
```

Comment:
For architecture tests it usually does not make sense to create folders per module, it can be all in root directory